react-native-git-upgrade/cliEntry.js

```diff
@@ -294,8 +315,12 @@ async function run(requestedVersion, cliArgs) {
     log.error('An error occurred during upgrade:');
     log.error(err.stack);
     if (projectBackupCreated) {
-      log.error('Restore initial sources');
+      log.info('Restore initial sources');
       await exec('git checkout project-snapshot', true);
+      if (oldReactPeerDepVersion) {
+        log.info('Go back to initial React version');
+        await exec('npm install --save react@' + oldReactPeerDepVersion + ' --color=always', verbose);
```

Comment:
Interesting point, it's a good idea to restore the initial node_modules tree. But we don't need to manage a reference toward the initial version.

The `git checkout project-snapshot` restore the original `package.json` file, so just run `npm install` here, the initial version of R **and RN** will be restored 💪